dist/src/main/java/io/zeebe/broker/StandaloneBroker.java

```diff
@@ -28,6 +31,7 @@
 @SpringBootApplication(exclude = ElasticsearchRestClientAutoConfiguration.class)
 @ComponentScan({"io.zeebe.broker", "io.zeebe.shared"})
 public class StandaloneBroker implements CommandLineRunner {
+  private static final Logger LOG = LoggerFactory.getLogger(StandaloneBroker.class);
```

Comment:
Shouldn't we use the existing `SystemLogger`? https://github.com/zeebe-io/zeebe/blob/develop/broker/src/main/java/io/zeebe/broker/Loggers.java#L16

Reply:
Should we? It's unclear to me what is referred to here by the system, to be honest. I think this kind of fits with the point I was trying to make above, but again, outside of scope for this PR, so let's go with that for now.